casbin/enforcer.py

```diff
@@ -271,7 +271,7 @@ def enforce(self, *rvals):
             for j, token in enumerate(self.model.model["r"]["r"].tokens):
```

Comment:
I think this is reasonable. Maybe ``for j, token in enumerate(self.model.model["r"]["r"].tokens):`` should also be changed? What do you think @techoner ?

Reply:
I approved these changes he made.